### PR TITLE
Adds list of kn plugins to docs

### LINKED ISSUE #4719: Add a list of available kn plugins
Add a list of plugins to the kn plugins page: https://knative.dev/docs/install/client/kn-plugins/

### PR DESCRIPTION
**Pull request by psschwei**
## Proposed Changes 

Adds list of kn plugins and some details around installing them. Any suggestions welcome :) 

Fixes #4719

### CODE REVIEW

File: docs/install/client/kn-plugins.md [hunk 1]
```diff
@@ -5,4 +5,30 @@ The `kn` CLI supports the use of plugins. Plugins enable you to extend the funct
...
+
+## How to install a plugin
+
+### Manaul installation
+
+All plugins can be installed by downloading the current release from Github and placing it somewhere on your `PATH`. You may need to rename the file to remove any OS/architecture information (for example, `kn-admin-darwin-amd64` should be renamed to `kn-admin`).
+
+### Homebrew
+
+Some plugins can be installed via the [Knative plugins Homebrew Tap](https://github.com/knative-sandbox/homebrew-kn-plugins/)) For example, the `kn-admin` plugin can be installed by running `brew install knative-sandbox/kn-plugins/admin`.
```
Comment (snneji) [inline docs/install/client/kn-plugins.md:18]:
```suggestion
You can install some plugins can be installed using the [Knative plugins Homebrew Tap](https://github.com/knative-sandbox/homebrew-kn-plugins/). For example, you can install the `kn-admin` plugin by running `brew install knative-sandbox/kn-plugins/admin`.
```

File: docs/install/client/kn-plugins.md [hunk 2]
```diff
@@ -5,4 +5,30 @@ The `kn` CLI supports the use of plugins. Plugins enable you to extend the funct
...
+
+## List of Knative plugins
+
+| Plugin | Description | Available via Homebrew? |
+| --- | --- | :---: |
+| [kn-plugin-admin](https://github.com/knative-sandbox/kn-plugin-admin) | Kn plugin for managing a Kubernetes based Knative installation | Y |
```
Comment (snneji) [inline docs/install/client/kn-plugins.md:24]: I think `kn` should be lower case and in code to be consistent.

**Review by snneji — CHANGES_REQUESTED** [review]
Thanks for doing this one @psschwei -- the table looks really good! 🙂 

I have just a few questions and suggestions to add.

File: docs/install/client/kn-plugins.md [hunk 3]
```diff
@@ -5,4 +5,30 @@ The `kn` CLI supports the use of plugins. Plugins enable you to extend the funct
...
     The plugins must be named with the prefix `kn-` to be detected by `kn`. For example, `kn-func` will be detected but `func` won't be detected.
 
-<!--TODO: Add details about installing and using different available plugins, list of available plugins-->
+<!--TODO: Add details about using different available plugins-->
+
+## How to install a plugin
+
+### Manaul installation
+
+All plugins can be installed by downloading the current release from Github and placing it somewhere on your `PATH`. You may need to rename the file to remove any OS/architecture information (for example, `kn-admin-darwin-amd64` should be renamed to `kn-admin`).
```
[the remark this answers is not in the document]
Comment (psschwei) [reply docs/install/client/kn-plugins.md:14]: +1 on making executable before moving.

As for the installation location, how about making the last instruction something like:
`Move the file to either a directory on your `PATH` or to the [`plugins/` directory](configure-kn.md) for your operating system.`?

[the remark this answers is not in the document]
Comment (snneji) [reply docs/install/client/kn-plugins.md]: My only concern with giving 2 options is that uses might not know which to choose. Is there a best practice? If so, that's what we should document.

[the remark this answers is not in the document]
Comment (psschwei) [reply docs/install/client/kn-plugins.md]: Got it. I think PATH is best practice, so will update to just include that option.

**Review by snneji — CHANGES_REQUESTED** [review]
Sorry, one more change to fix a link. Once you've applied the change ping me on Slack and I'll apply the LGTM.

**Pull request merged**